src/commands/admin/announce.ts

```diff
@@ -0,0 +1,30 @@
+import { botMasterPerms } from '@lib/permissions';
+import { channelParser } from '@lib/arguments';
+import { TextChannel, Message } from 'discord.js';
+import { CHANNELS } from '@root/config';
+
+export const description = 'Sends an announcement from Sage to a specified channel or announcements if no channel is given.';
+export const usage = '[channel]|<content>';
+
+export async function permissions(msg: Message): Promise<boolean> {
+	return await botMasterPerms(msg);
+}
+
+export async function run(_msg: Message, [channel, content]: [TextChannel, string]): Promise<Message> {
+	return channel.send(content);
```

Comment:
Maybe also send a message in the msg.channel saying that the announcment has been sent.